src/contracts/vendored/mixins/BridgedTokensRegistry.sol

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+// Vendored from omnibridge, see:
+// <https://github.com/omni/omnibridge/blob/b658c7c217e25c13e61ab9fb1a97010a5656b11e/contracts/upgradeable_contracts/components/bridged/BridgedTokensRegistry.sol>
```

Comment:
mac can not deal with the `(` and the -1 index:
```

./bash.sh: line 3: -1: substring expression < 0
./bash.sh: line 4: syntax error near unexpected token `('
./bash.sh: line 4: `diff <(curl -s "${vendored_url:1:-1}" | npx prettier --parser solidity-parse) "$contract"'
```

Reply:
Sorry Alex, I should probably have tried to ask somebody with a Mac to try the test plan before now.
To help out, I made the necessary changes in #66. Note that the contract files from the URL are parsed _after_ the diff, so it would need to be formatted before being added. This is done because editors have the tendency to autoformat code, so we don't have to worry about making changes and then undo any autoformatting.

In any case, I think the error in line 4 might be the same as in line 3, then this might work:
```
contract='src/contracts/vendored/mixins/BridgedTokensRegistry.sol'
bracketed_url="$(grep -o '<https://.*>' "$contract" | head -n 1)"
vendored_url="$(echo -n "$bracketed_url" | head -c -1 | tail -c +2)"
echo "Diff for $contract, vendored from $vendored_url"
diff <(curl -s "$vendored_url" | npx prettier --parser solidity-parse) "$contract"
```